src/jumio-kyc/kyc.service.ts

```diff
@@ -0,0 +1,81 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { Injectable, InternalServerErrorException } from '@nestjs/common';
+import { KycStatus, Redemption, User } from '@prisma/client';
+import { JumioApiService } from '../jumio-api/jumio-api.service';
+import { JumioTransactionService } from '../jumio-transactions/jumio-transaction.service';
+import { PrismaService } from '../prisma/prisma.service';
+import { BasePrismaClient } from '../prisma/types/base-prisma-client';
+import { RedemptionService } from '../redemptions/redemption.service';
+
+export type KycDetails = {
+  jumio_account_id: string;
+  jumio_workflow_execution_id: string;
+  jumio_web_href: string;
+};
+
+@Injectable()
+export class KycService {
+  constructor(
+    private readonly prisma: PrismaService,
+    private readonly redemptionService: RedemptionService,
+    private readonly jumioTransactionService: JumioTransactionService,
+    private readonly jumioApiService: JumioApiService,
+  ) {}
+
```

Comment:
This region of the code is simplified into a single method in subsequent PR. When I was implementing API interaction with Jumio (PR 3/3) I discovered we could simplify.